docs/README.md

```diff
@@ -35,9 +35,9 @@ Review the [Change Log](ChangeLog.md) page for a history of changes in each rele
 
 ## Prerequisites for the Splunk Operator
 
-### Supported Kubernetes Versions
+## Platform recommendations
```

Comment:
Nit: should this have three #'s for indentation purposes?

Reply:
fixed